README.md

```diff
@@ -1,33 +1,147 @@
 
-In your Storyblok space, go to **Settings** > **Visual Editor** and set `https://localhost:3000/` as the default environment.
+To learn more about Astro's file structure, refer to [the official guide on project structure](https://docs.astro.build/en/basics/project-structure/).
+
+## Commands
+
+The blueprint supports the following commands. Run these from the root of the project:
+
+| Command                | Action                                                                           |
+| :--------------------- | :------------------------------------------------------------------------------- |
+| `npm install`          | Installs dependencies                                                            |
+| `npm run storyblok`    | Starts a local development server with HTTPS enabled at `https://localhost:3000` |
+| `npm run dev`          | Starts a local dev server at `localhost:4321`                                    |
+| `npm run build`        | Build your production site to `./dist/`                                          |
+| `npm run preview`      | Preview your build locally, before deploying                                     |
+| `npm run astro ...`    | Run CLI commands like `astro add` or `astro check`                               |
+| `npm run astro --help` | Get help using the Astro CLI                                                     |
+
+## Learn more
+
+To learn more about Storyblok, visit [Storyblok Documentation](https://www.storyblok.com/docs).
 
-Finally, open your **Home** story and see your Next.js project running correctly. Happy building!
+To learn more about Astro, check the [official documentation](https://docs.astro.build/en/getting-started/).
 
-## Learn More
+To learn more about the integration between Storyblok and Astro, check out our [dedicated developer tutorials](https://www.storyblok.com/tutorials?technologies=astro).
 
-To learn more about Next.js, take a look at the following resources:
+***
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+> [!TODO]
+> - [ ] Add GitHub Codespaces [badge](https://github.com/codespaces/badge.svg) ([Prebuilding your codespaces - GitHub Docs](https://docs.github.com/en/codespaces/prebuilding-your-codespaces))
+> - [ ] Add link to Discord + badge [![Discord Chat](https://img.shields.io/discord/308323056592486420.svg)](https://discord.gg/)  
+> - [ ] Add link to [docs](https://www.storyblok.com/docs)
+> - [ ] Add link to [Help Center](https://support.storyblok.com/hc/en-us)
+> - [ ] Add link to the specific technology guide
+> - [ ] Add link to [sign up](https://app.storyblok.com/#/signup)
+> - [ ] List packages and versions: Astro [5.7.1](https://github.com/withastro/astro/releases/tag/astro%405.7.1), SDK [6.1.1](https://github.com/storyblok/storyblok-astro/releases/tag/v6.1.1), Vite [6.3.3](https://github.com/vitejs/vite/releases/tag/v6.3.3)
```

Comment:
Isn't that already provided by the package.json? This could easily be forgotten when updating/maintaing the repository.